examples/android/android-studio/Cukeulator/README.md

```diff
@@ -0,0 +1,72 @@
+## Cukeulator Example Test
+This is the example test-project for the Cukeulator app for Android Studio (Beta) 0.8.14.
+
+### Setup
+Features must be placed in `assets/features/`. Subdirectories are allowed.
+
+The rest of the dependencies are added automatically in `app/build.gradle`.
+
+The cucumber-android dependency is added as:
+
+```
+androidTestCompile 'info.cukes:cucumber-android:1.2.0@jar'
```

Comment:
@SierraGolf No, I don't know why the cucumber-android module was specified to be packaged as an apklib. But as the apklib is an [maven-android-plugin invented format](https://github.com/cucumber/cucumber-jvm/issues/573#issuecomment-23276981), could it be that the apklib at the time that cucumber-android was [originally created](https://github.com/cucumber/cucumber-jvm/pull/525), it was the packaging that was promoted for an Android library project by the maven-android-plugin team?